.travis.yml

```
@@ -53,23 +54,18 @@ install:
   - npm install
   - bundle install --without development production
 before_script:
-  - "export DISPLAY=:99.0"
-  - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1600x1200x16"
-  - sleep 3
+  - echo ===[TEST_WORKER=$TEST_WORKER before_script $(date -u +%Y%m%d-%H%M%S)]===
+  - cp config/application_settings.yml.travis config/application_settings.yml
   - cp config/database.yml.travis config/database.yml
   - cp config/secrets.yml.example config/secrets.yml
-  - mkdir tmp/
-  - mkdir tmp/downloads
+  - mkdir -p tmp/downloads
   - bundle exec rake db:create RAILS_ENV=test # database user by default is `travis` 
   - bundle exec rake db:migrate RAILS_ENV=test
-  - ls -alh bin 
-  - ls -alh public/packs-test 
 script:
-  - bundle exec rake assets:precompile
-# - bundle exec parallel_test spec/ -n 6 --only-group $TEST_GROUP --group-by filesize --type rspec
-  - bundle exec rspec
+  - .travis/spec_runner.sh
 notifications:
   email:
     recipients:
       - diapriid@gmail.com
```

Should we add others here? @LocoDelAssembly?

---

@mjy I guess I could add myself there.
```suggestion
      - ellocodelassembler@gmail.com
```